packages/shortcuts-extension/src/components/ShortcutList.tsx

```diff
@@ -36,10 +46,17 @@ export class ShortcutList extends React.Component<IShortcutListProps> {
         }}
         id="shortcutListContainer"
       >
-        <div className="jp-Shortcuts-ShortcutList">
+        <div
+          className="jp-Shortcuts-ShortcutList"
+          id="Shortcuts-ShortcutList"
```

Comment:
Please do not add an identifier here. There can be multiple shortcut lists in principle.

```suggestion
```